assets/semgrep_rules/client/const_cast.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+  - id: const_cast
+    metadata:
+      author: Brian Johnson
+      references:
+        - https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++-dos-and-donts.md#use-correctly
+      source: https://github.com/brave/security-action/blob/main/assets/semgrep_rules/client/const_cast.yaml
+      assignees: |
+        bridiver
+        cdesouza-chromium
+    languages: [cpp]
+    message: "Avoid const_cast to remove const, except when implementing non-const getters in terms of const getters."
```

Comment:
double quotes not required

Reply:
oops, I copied the template so I guess original one had quotes